readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://LBondarchuk.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://LBondarchuk.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Your test report link doesn't work. Did you run tests? 
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/99403198/216316945-c17ae2b5-01ec-423d-86fa-2e1440ee915f.png">